docs/topics/js/js-to-kotlin-interop.md

```diff
@@ -100,9 +100,10 @@ external fun newC()
 
 ### @JsExport annotation
 
-> The `@JsExport` annotation is currently marked as experimental. Its design may change in future versions.
+> This feature is [Experimental](components-stability.md#stability-levels-explained).
```

Comment:
As this is experimental, do we have a YT issue for feedback as we do with @JsStatic?